sklearn/model_selection/tests/test_search.py

```diff
@@ -152,7 +155,8 @@ def test_grid_search():
     assert_equal(grid_search.best_estimator_.foo_param, 2)
 
     for i, foo_i in enumerate([1, 2, 3]):
-        assert_true(grid_search.grid_scores_[i][0] == {'foo_param': foo_i})
```

Comment:
We just have the `grid_scores_` for backward compat no? Moreover I've tested it to ensure it remains the same as `results_`...